.github/workflows/create-tests-matrices/action.yml

```diff
@@ -0,0 +1,48 @@
+inputs:
+    language-name:
+        description: "Language name"
+        required: true
+        default: false
+outputs:
+    should-run:
+        description: "Should run the job"
+        value: ${{ steps.check_should_run.outputs.should-run }}
+    engine-matrix-output:
+        description: "Engine matrix"
+        value: ${{ steps.load-engine-matrix.outputs.engine-matrix }}
+    host-matrix-output:
+        description: "Host matrix"
+        value: ${{ steps.load-host-matrix.outputs.host-matrix }}
+
+runs:
+    using: "composite"
+    steps:
+        - name: Check for Core changes label
+          id: check_should_run
+          shell: bash
+          run: |
+              if [[ conditions ]]; then
```

Comment:
What is `conditions`? It is not defined.

Reply:
That's a mistake, I'll fix